generated/java/build.gradle

```diff
@@ -31,8 +31,9 @@ subprojects {
   ext {
     // Shortcuts for libraries we are using
     libraries = [
-        gax: 'com.google.api:gax:0.14.0',
-        gaxGrpc: 'com.google.api:gax-grpc:0.14.0',
+        gax: 'com.google.api:gax:1.1.0',
+        gaxGrpc: 'com.google.api:gax-grpc:0.17.0',
+        iam: 'com.google.api.grpc:grpc-google-iam-v1:0.1.9',
```

Comment:
There was an issue of batch generation config that iam was not correctly staged. I will manually update this.